scripts/world_bank/wdi/manifest.json

```diff
@@ -0,0 +1,22 @@
+{
+    "import_specifications": [
+        {
+            "import_name": "WorldDevelopmentIndicators",
+            "curator_emails": [
+                "sanikap@google.com"
+            ],
+            "provenance_url": "https://datacatalog.worldbank.org/dataset/world-development-indicators/",
+            "provenance_description": "Variables related to demographics, energy, health, labor, etc. from the World Bank",
+            "scripts": [
+                "worldbank.py"
```

Comment:
The "indicatorSchemaFile" flag is marked as required in the script, but here it is not passed. Could you recheck this?

Reply:
Default path is provided for indicatorSchemaFile